themes/schema.json

```diff
@@ -185,6 +185,15 @@
               "description": "https://ohmyposh.dev/docs/configure#postfix",
               "default": " "
             },
+            "include_folders": {
+              "type": "array",
+              "title": "If specified, segment will only render in these folders",
+              "description": "https://ohmyposh.dev/docs/configure#include-folders",
+              "default": [],
+              "items": {
+                "type": "string"
+              }
+            },
             "ignore_folders": {
```

Comment:
Should we rename ignore_folders to exclud_folders. Seems more logical to me but introduces a kind of breaking change.